tests/js/server/replication/sync/replication-sync-malarkey.js

```diff
@@ -895,6 +895,47 @@ function BaseTestConfig () {
       
       checkCountConsistency(cn, expected);
     }, 
+    // create large AQL operation on leader using AQL
+    testIncrementalQuickKeys: function () {
```

Comment:
This test will run 3 times now, once for the old protocol, and twice for the new protocol (Where it won't have any effect). We could move it to the _OldFormat test suite, so we save a bit of test runtime.

Reply:
We should in addition have a dedicated test for the HTTP API, so we verify that the "quick" parameter actually does what it is expected to do.

This should be trivial from the arangosh:
We can first set the failure point on the leader as we do here, and then call the API for collections that have more, less or equal amounts of documents than 100. In each case, the response should match the expectations. This should be a separate test in `tests/js/client/shell`, and should work in single server at least (can also be made to work on the cluster, but will be a bit more work).